src/content/docs/synthetics/synthetic-monitoring/using-monitors/device-emulation.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Device emulation
+tags:
+  - Synthetics
+  - Synthetic monitoring
+  - Using monitors
+translate:
```

Comment:
@bpeckNR You can remove this translate frontmatter section since all docs are now translated into Japanese.

Reply:
Done